integration-tests/testkit/seed.ts

```diff
@@ -708,6 +695,22 @@ export function initSeed() {
 
                   return result.target?.schemaVersions.edges.map(edge => edge.node);
                 },
+                async updateSchemaVersionStatus(
```

Comment:
It was never intended to run this via a registry access token, so I moved it to the project level and execute it with the organization owner token instead.